bot/adapters/rest/add_user_standalone.py

```diff
@@ -0,0 +1,222 @@
+import asyncio
+import os
+import sys
+from datetime import date, timedelta
+from pathlib import Path
+from typing import Optional, Dict, Any
+
+try:
+    import asyncpg
+except ImportError:
+    print("❌ Error: asyncpg not installed. Install with: pip install asyncpg")
+    sys.exit(1)
+
+try:
+    import bcrypt
+except ImportError:
+    print("❌ Error: bcrypt not installed. Install with: pip install bcrypt")
+    sys.exit(1)
+
+
+class UserManager:
+    def __init__(self):
+        self.load_env_file()
+        self.db_config = self._get_db_config()
+
+    def load_env_file(self) -> None:
+        env_file = Path(__file__).parent / '.env'
+        if not env_file.exists():
+            return
+
+        with open(env_file, 'r', encoding='utf-8') as f:
+            for line in f:
+                line = line.strip()
+                if self._is_valid_env_line(line):
+                    key, value = line.split('=', 1)
+                    if key not in os.environ:
+                        os.environ[key] = value.strip()
+
+    @staticmethod
+    def _is_valid_env_line(line: str) -> bool:
+        return line and not line.startswith('#') and '=' in line
+
+    @staticmethod
+    def _get_db_config() -> Dict[str, Any]:
+        return {
+            'host': os.getenv('POSTGRES_HOST', '192.168.1.210'),
+            'port': int(os.getenv('POSTGRES_PORT', '30665')),
+            'database': os.getenv('POSTGRES_DB', 'postgres'),
+            'user': os.getenv('POSTGRES_USER', 'RanchBot'),
+            'password': os.getenv('POSTGRES_PASSWORD', ''),
+            'server_settings': {
+                'search_path': os.getenv('POSTGRES_SCHEMA', 'ranczo')
+            }
+        }
+
+    @staticmethod
+    def hash_password(password: str) -> str:
+        salt = bcrypt.gensalt()
+        return bcrypt.hashpw(password.encode('utf-8'), salt).decode('utf-8')
+
+    async def connect_to_db(self) -> asyncpg.Connection:
+        schema = self.db_config['server_settings']['search_path']
+        print(f"🔌 Connecting to PostgreSQL: {self.db_config['user']}@{self.db_config['host']}:"
+              f"{self.db_config['port']}/{self.db_config['database']} (schema: {schema})")
+
+        try:
+            return await asyncpg.connect(**self.db_config)
+        except (OSError, asyncpg.PostgresError) as e:
+            print(f"❌ Connection failed: {e}")
+            sys.exit(1)
```

Comment:
ale skoro to jest standalone, to po co cyrkować z sysexit? nie lepiej pozwolić temu errorowi ubić appkę po prostu?

Ewentualnie, zostaw se te printy i daj `raise` albo `raise e` na koniec

Reply:
to do wszystkich sys.exit(1) jak cos